lib/zendesk_api/configuration.rb

```diff
@@ -55,6 +55,9 @@ def options
           :accept_encoding => 'gzip;q=1.0,deflate;q=0.6,identity;q=0.3',
           :user_agent => "ZendeskAPI API #{ZendeskAPI::VERSION}"
         },
+        :request => {
+          open_timeout: 10
```

Comment:
Mind keeping the hash rocket? 1.8 isn't officially supported, but it is "supported" as far as I know